stl/inc/xstring

```diff
@@ -1240,9 +1240,9 @@ public:
 
 #ifdef __cpp_lib_concepts
     // clang-format off
-    template <contiguous_iterator _It, sized_sentinel_for<_It> _Se>
-        requires (is_same_v<iter_value_t<_It>, _Elem> && !is_convertible_v<_Se, size_type>)
-    constexpr basic_string_view(_It _First, _Se _Last) noexcept(noexcept(_Last - _First)) // strengthened
+    template <contiguous_iterator _Iter, sized_sentinel_for<_Iter> _Sent>

```

Comment:
I'm not against the renaming, but why make it a part of such a massive PR? (same below)

Reply:
miscco was bothered that there were both Ranges-style `<_It, _Se>` function templates and non-Ranges-style `<_Iter>` function templates in the same file. I made all of them consistent.